src/main/java/eu/europeana/metis/sandbox/service/validation/ValidationServiceConfigImpl.java

```diff
@@ -0,0 +1,17 @@
+package eu.europeana.metis.sandbox.service.validation;
+
+import eu.europeana.validation.service.ValidationServiceConfig;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+class ValidationServiceConfigImpl implements ValidationServiceConfig {
```

Comment:
Perhaps this is a place where more validation configuration options could be added (see other remarks)?

Reply:
This one is a `@Service` (but yes the name does not help) it was a requirement of the `ValidationExecutionService` that is in metis-validation-service. (I followed the approach used in metis-validation-rest which injects `ValidationExecutionService` in `ValidationController`)

I am using `ValidationExecutionService` in the `ExternalValidationServiceImpl` but to be able to inject that service I needed to provide the beans that it depends on, which are
`ValidationServiceConfig validationServiceConfig` which is this implementation
`ClasspathResourceResolver lsResourceResolver` bean defined in configuration file
`SchemaProvider schemaProvider` bean defined in configuration file

Reply:
Well, it seems to be a class that reads properties (in fact only one property) from the configuration file and return it. So I thought that this would be a place to add similar code for the properties that the validation code needs and pass that along explicitly. But it seems you found a way to do it implicitly. I will remark further on it elsewhere.